opentelemetry-api/src/opentelemetry/trace/span.py

```diff
@@ -390,6 +390,26 @@ def values(self) -> typing.ValuesView[str]:
 
 DEFAULT_TRACE_STATE = TraceState.get_default()
 
+_TRACE_ID_BYTES = 16
+_TRACE_ID_HEX_LENGTH = 2 * _TRACE_ID_BYTES
+
+
+def _validate_trace_id(trace_id: int) -> bool:
+    """Validates a trace_id.
+
+    Args:
+        trace_id: An int that is expected to corresponds to 16-bytes hexdecimal value
+
+    Returns:
+        True if trace_id is valid, False otherwise.
+    """
+    if not trace_id:
+        return False
+    if len(format(trace_id, "032x")) != _TRACE_ID_HEX_LENGTH:
```

Comment:
Would this work and be a bit faster?

```py
# constant somewhere
_MAX_TRACE_ID = (1 << 128) - 1
```

```suggestion
    if trace_id > _MAX_TRACE_ID:
```

Reply:
Yes, I also prefer to make sure the trace id value is lesser than a certain value. Also, instead of `(1 << 128) - 1` we can do `2 ** 128 - 1` which is subjectively easier to understand.